### ISSUE TITLE
 conditional import against six.PY2 emits error

### ISSUE THREAD

**Issue by georgexsh** [pylint-dev/pylint#3501]
### Steps to reproduce
1. Given following code
```
import six
if six.PY3:
    from urllib.parse import urlparse
else:
    from urlparse import urlparse
```
2. run pylint with python2

### Current behavior

```
[E0611(no-name-in-module), ] No name 'parse' in module 'urllib'
```

### Expected behavior
no error reported

### pylint --version output
```
Using config file .pylintrc
pylint 1.9.5,
astroid 1.6.5
Python 2.7.16 (default, Jul 12 2019, 01:42:49)
[GCC 6.3.0 20170516]
```

ref: https://github.com/PyCQA/pylint/issues/769

**Comment by PCManticore** [comment]
Thanks for the report. The original #769 issue was related to fallback imports, but we should support `six` as well.

**Comment by LexiconCode** [comment]
Results as follows:

python 2 windows (2.7.x) Linting Fail 
- E:  7, 4: Unable to import 'tkinter' (import-error)

python 3 windows (3.8.x) Success Linting

python 3 linux (3.8.x) Linting Fail 
- E:  7, 4: Unable to import 'tkinter' (import-error)

python 3 MacOs (3.8.x)  Success Linting

Setup:
Github Actions Configuration
https://0bin.net/paste/sGbbX84r#vxpe73nMg6-YHd+Y+Z+hqCl8YRh9a8pmzgMK01PfWeF

tk_test.py
```python
import sys, six
print(sys.version)

if six.PY2:
    import Tkinter as tk
else:
    import tkinter as tk

print(tk.TkVersion) # succeeds in all platforms to print version number
```

**Comment by Pierre-Sassoulas** [comment]
Closing as it's related to six / python 3 porting so unlikely to get attention, and also because the generic issue seems related to control-flow which has dozen of open issues already.
